### ISSUE TITLE
Question.

### ISSUE THREAD

**Issue by dogpacolypse100** [TheNightmareX/pennfoster-helper#2]
I know this may seem irrelevant but I've never executed a GitHub repo, is there any way I can find out how to? Or perhaps just this repo?

**Comment by TheNightmareX** [comment]
It's different for every repository.

By "execute this repository" I think you mean to use this chrome extention.
I have already compiled the sources and uploaded the output files as a zip attachment [pennfoster-helper.zip](https://github.com/TheNightmareX/pennfoster-helper/releases/download/v0.1.0/pennfoster-helper.zip) of the [release](https://github.com/TheNightmareX/pennfoster-helper/releases/tag/v0.1.0).
To install this zipped chrome extension to your browser, follow any guides from the internet such as https://www.theseotoday.com/2022/09/how-to-add-extensions-to-chrome.html

**Comment by dogpacolypse100** [comment]
> It's different for every repository.
> 
> 
> 
> By "execute this repository" I think you mean to use this chrome extention.
> 
> I have already compiled the sources and uploaded the output files as a zip attachment [pennfoster-helper.zip](https://github.com/TheNightmareX/pennfoster-helper/releases/download/v0.1.0/pennfoster-helper.zip) of the [release](https://github.com/TheNightmareX/pennfoster-helper/releases/tag/v0.1.0).
> 
> To install this zipped chrome extension to your browser, follow any guides from the internet such as https://www.theseotoday.com/2022/09/how-to-add-extensions-to-chrome.html

Yes, thank you very much (I'm a noob at this stuff) but I very much appreciate your assistance!

**Comment by Char2sGu** [comment]
> Does this still work?

It will work unless PennFoster has updated their exam system, which is likely not possible.
The extension is still there so you can easily try it out!

**Comment by MisterCi** [comment]
when i try to put the extension in chrome as the guide showed, i cant find the folder in which to drop into chrome.

**Comment by Char2sGu** [comment]
@MisterCi You need to unzip the `pennfoster-helper.zip` archive first. The folder is wherever you chose to unzip it.

**Comment by MisterCi** [comment]
@Char2sGu now it says this Failed to load extension
File
~\Downloads\pennfoster-helper-main
Error
Manifest file is missing or unreadable
Could not load manifest.


Lol

**Comment by Char2sGu** [comment]
@MisterCi 
Please download the compiled assets instead of the source code.
Please check my first reply in this ticket.
